src/trans-netlist/trans_to_netlist.cpp

```diff
@@ -303,6 +308,62 @@ void convert_trans_to_netlistt::operator()(const irep_idt &module)
   dest.initial.push_back(instantiate_convert(
     aig_prop, dest.var_map, trans.init(), ns, get_message_handler()));
 
+  // properties
+  for(const auto &[id, symbol] : symbol_table)
+  {
+    if(symbol.module == module && symbol.is_property)
+    {
+      auto expr = normalize_property(symbol.value);
+
+      auto convert = [&aig_prop, this](const exprt &expr) -> literalt {
+        return instantiate_convert(
+          aig_prop, dest.var_map, expr, ns, get_message_handler());
+      };
+
+      if(expr.id() == ID_AG || expr.id() == ID_G || expr.id() == ID_sva_always)
+      {
+        auto get_phi = [](const exprt &expr) {
+          if(expr.id() == ID_AG)
+            return to_AG_expr(expr).op();
+          else if(expr.id() == ID_G)
+            return to_G_expr(expr).op();
+          else if(expr.id() == ID_sva_always)
+            return to_sva_always_expr(expr).op();
+          else
+            PRECONDITION(false);
+        };
+
+        auto phi = get_phi(expr);
+
+        if(!has_temporal_operator(phi))
+        {
+          dest.properties.emplace(id, netlistt::Gpt{convert(phi)});
+        }
+        else if(
+          phi.id() == ID_AF || phi.id() == ID_F ||
+          phi.id() == ID_sva_s_eventually)
+        {
+          auto get_psi = [](const exprt &expr) {
+            if(expr.id() == ID_AF)
+              return to_AF_expr(expr).op();
+            else if(expr.id() == ID_F)
+              return to_F_expr(expr).op();
+            else if(expr.id() == ID_sva_s_eventually)
+              return to_sva_s_eventually_expr(expr).op();
+            else
+              PRECONDITION(false);
+          };
+
+          dest.properties.emplace(id, netlistt::GFpt{convert(get_psi(phi))});
+        }
+        else
+        {
+          // unsupported
```

Comment:
Should this be `UNIMPLEMENTED;`?

Reply:
No, I'd prefer this to produce a nice user-facing message in the status output, as opposed to a stack trace.